### ISSUE TITLE
FunctionTransformer is not supported unless the transform function is None (= identity)

### ISSUE THREAD

**Issue by paranjapeved15** [onnx/sklearn-onnx#1040]
**Hi, I am trying to use the following code to calculate "c" from a and b.** 

  ```
def calculate_overprice(data, col1, col2):
      result = np.float32(100) * (data[col1] - data[col2]) / data[col2]
      return pd.DataFrame(result,columns=["c"])

  mapper = ColumnTransformer(
          transformers=[
              (
                  "c",
                  FunctionTransformer(func=calculate_overprice, kw_args={'col1': 'a', 'col2': 'b'}),
                  ['a', 'b']
              )
          ],
          remainder='passthrough',
          verbose_feature_names_out=False
      )
      mapper.set_output(transform="pandas")
    
    pipe = Pipeline([("mapper", mapper), ("classifier", model_pkl)])
    
  convert_sklearn(
          pipe,
          "pipeline_xgboost",
          initial_types=initial_types,
          target_opset=14
      )
```
      
 **But it gives me the following error** 
      FunctionTransformer is not supported unless the transform function is None (= identity). 
      
    PS: When FunctionTransformer is listed as supported function here https://onnx.ai/sklearn-onnx/supported.html

**Comment by xadupre** [comment]
Comment posted on issue #609.

skl2onnx converts a scikit-learn pipeline into onnx. It means for many estimators, skl2onnx has an onnx implementation of the inference function implemented in scikit-learn. The final onnx graph is just putting all these blocks together. 

FunctionTransformer includes a piece of code skl2onnx has no implementation for and one piece is missing unless the user provides that implementation.

Now how to provide that onnx implementation of your custom code. That's the main difficult part. The blog post you mention uses a package I don't maintain anymore as I broke it into smaller packages. Some parts were added to onnx. I tried to make a list of the options available today: [Many ways to implement a custom graph in ONNX](https://sdpython.github.io/doc/onnx-array-api/dev/tutorial/onnx_api.html). In your case, the first choice is to select the option which fits your need.

Once you have an onnx representation of your python code, I recently added function `add_onnx_graph` in PR https://github.com/onnx/sklearn-onnx/pull/1023 which integrates any onnx graph into your pipeline. It is not released yet but skl2onnx can be installed from github to use it.
